dub.json

```
@@ -43,7 +43,10 @@
             "name": "server",
             "targetName": "agora",
             "mainSourceFile": "source/agora/node/main.d",
-            "excludedSourceFiles": [ "source/agora/cli/client/*" ]
+            "excludedSourceFiles": [
+                "source/agora/cli/client/*",
+                "source/agora/cli/multi/*"
+            ]
```

```suggestion
            "excludedSourceFiles": [ "source/agora/cli/*" ],
```

If it doesn't work, the following:
```suggestion
            "excludedSourceFiles": [ "source/agora/cli/*/*" ],
```